viewer/pom.xml

```diff
@@ -314,6 +314,7 @@
                                 <param>${viewerhtml.path}/common/ajax/EditFeature.js</param>
                                 <param>${viewerhtml.path}/common/ajax/ArcQueryUtil.js</param>
                                 <param>${viewerhtml.path}/common/ajax/FeatureExtent.js</param>
+                                <param>${viewerhtml.path}/common/ajax/File.js</param>
```

Comment:
zit deze file er wel in? zie 'm niet in deze PR, dat zou een probleem zijn voor de closure compiler

Reply:
ik zie `File.js` niet... 💣